app/controllers/change_item_types_controller.rb

```diff
@@ -9,8 +9,8 @@ def create
     @change_item_type = ChangeItemType.new(params[:change_item_type])
     if @change_item_type.valid?
       array_of_item_ids = @change_item_type.parse_uploaded_file
-      UniUpdates.where(item_id: array_of_item_ids).update_all(curr_lib: @change_item_type.current_library,
-                                                              new_itype: @change_item_type.new_item_type)
+      @uni_updates_batch = UniUpdatesBatch.create_item_type_batch(params)
```

Comment:
You may want to avoid using instance variables (`@variable`) in controllers unless you plan on using that instance variable in the view layer rendered by the controller action.  I haven't looked at the entire codebase, but it doesn't look like `@uni_updates_batch` or `@uni_updates` is used anywhere else besides the controller action itself.

FWIW, a good rule of thumb for a controller action is to only have one instance variable so that the view layer is not cluttered with accessing various instance variables.